### PR TITLE
[make:user] Use password_hashers instead of encoders

### PR DESCRIPTION
**Pull request by wouterj**
And also improve the position of this config when "enable_authenticator_manager" is the first option.

@jrushlow can you maybe help with the blank line (see failing tests)? I tried some things (including adding a `'_' => $this->manipulator->createEmptyLine()`) but all resulted in major failures. Thanks!

### CODE REVIEW

**Review by javiereguiluz — APPROVED** [review]

**Comment by wouterj** [comment]
There are all sorts of failures in the tests that I can't explain with my changes - I hope they aren't related.

Apart from that, this one should (finally) be ready.

**Review by jrushlow — APPROVED** [review]
Thanks @wouterj!

File: src/Security/SecurityConfigUpdater.php [hunk 1]
```diff
@@ -72,6 +74,10 @@ public function updateForAuthenticator(string $yamlSource, string $firewallName,
         $newData = $this->manipulator->getData();
 
         if (!isset($newData['security']['firewalls'])) {
+            if ($newData['security']) {
+                $newData['security']['_firewalls'] = $this->manipulator->createEmptyLine();
+            }
```
Comment (weaverryan) [inline src/Security/SecurityConfigUpdater.php:79]: What is this for? Are we suddenly missing an empty line somewhere?

**Review by weaverryan — APPROVED** [review]

**Comment by weaverryan** [comment]
This is quite thorough and complex - thank you Wouter!

**Pull request merged**